README.md

```diff
@@ -104,6 +104,11 @@ const pickr = Pickr.create({
         'rgba(255, 193, 7, 1)'
     ],
 
+    // either
+    save: true,
```

Comment:
Do you mean `components.interaction.save`? Because there is no root `save` option (I think)...